src/screens/MainScreen/index.js

```diff
@@ -1,22 +1,46 @@
-import React from 'react';
-import { View, Text, Button } from 'react-native';
-import MapView, { PROVIDER_GOOGLE } from 'react-native-maps';
+import React, { useRef } from 'react';
+import { View, Text, Image, TouchableOpacity, Animated, Dimensions } from 'react-native';
+import MapView, { PROVIDER_GOOGLE, Marker } from 'react-native-maps';
 
 import { MAIN_SCREEN } from 'constants/screens';
+import CreateTargetForm from 'components/CreateTargetForm';
+import TopicPicker from 'components/TopicPicker';
 import { LATITUDE_DELTA, LONGITUDE_DELTA } from 'constants/common';
-import useLogout from 'hooks/useLogout';
 import useLocation from 'hooks/useLocation';
 import useGetTargets from 'hooks/useGetTargets';
+import useCreateTarget from 'hooks/useCreateTarget';
 import strings from 'locale';
-import useSession from 'hooks/useSession';
 import CustomMarker from 'components/CustomMarker';
+import crosshair from 'assets/images/crosshair.png';
+import locationMap from 'assets/images/locationMap.png';
+import common from 'constants/commonStyles';
 import styles from './styles';
 
+const HIDDEN_VIEWS_POSITION = Dimensions.get('window').height * -1;
```

Comment:
You should have a constant for height and width window dimension since you will we using it in multiple places.